problem_builder/public/css/answer.css

```diff
@@ -1,6 +1,6 @@
 .mentoring .answer.editable {
     height: 250px;
-    width: 100%;
+    width: 99%;
```

Comment:
@itsjeyd the question is, why hadn't it worked like that before the changes? I would suggest investigating it - probably there's some regression added in this PR (or at least it doesn't work like before). This rule looks like it is always applied, so it is totally not an option to add it - in best case we'll have regressions coming from QA about "this used to be 100% width, but not anymore" or (more likely) "there's an additional horizontal spacing next to freeform answers". In any case, the fix for this should only be applied to studio preview views.

Idea: Add common_preview_view.css that is only added to preview view (at all levels and blocks) and apply this change there (probably will need a stronger css selector)